benchmarks/dynamo/common.py

```diff
@@ -2183,6 +2615,17 @@ def get_example_inputs(self):
     group.add_argument(
         "--xla", action="store_true", help="Compare TorchXLA to eager PyTorch"
     )
+    group.add_argument(
+        "--onnx",
```

Comment:
```suggestion
        "--torchscript-onnx",
```
Similar to the `OnnxModel` and `OnnxModelV2` comment, a more intuitive description would be nice. Although `--dynamo-onnx` is clear `--onnx` allows a lot of imagination, especially for those who are newer to pytorch and never met the torch script friend